yaml/boundaries.yaml

```diff
@@ -189,6 +213,22 @@ filters:
       <<: *output_properties
       kind: disputed_reference_line
       disputed: True
+      recognized_by: { col: recognized_by }
+      tz_admin_level: {col: admin_level }
+    table: osm
+  # These are kind=disputed_reference_lines that, for viewpoint XX in disputed_by,
+  # becomes kind:xx=unrecognized_disputed_reference_line
+  - filter: {admin_level: ['2','4','6'], boundary: disputed, geom_type: line}
+    min_zoom: 8
+    output:
+      <<: *output_properties
+      kind: disputed_reference_line
+      kind_detail: {col: admin_level }
+      disputed: True
+      disputed_by: {col: disputed_by }
```

Comment:
To make QA and data assertions easier, suggest exporting `brk_a3: {col: ne:brk_a3}`.

Reply:
You should also do this for the NE features in the 300s farther down.